rasa/core/broker.py

```diff
@@ -223,3 +225,52 @@ def _publish(self, event):
 
     def _close(self):
         self.producer.close()
+
+
+class SQLProducer(EventChannel):
+    from sqlalchemy.ext.declarative import declarative_base
+
+    Base = declarative_base()
+
+    class SQLEvent(Base):
+        from sqlalchemy import Column, Integer, String, Float, Text
+
+        __tablename__ = "events"
+        id = Column(Integer, primary_key=True)
+        sender_id = Column(String(255))
+        data = Column(Text)
+
+    def __init__(
+        self,
+        dialect: Text = "sqlite",
+        host: Optional[Text] = None,
+        port: Optional[int] = None,
+        db: Text = "events.db",
+        username: Text = None,
+        password: Text = None,
+        login_db: Optional[Text] = None,
```

Comment:
we should not accept a `login_db` paremeter, since we don't support the logic for it in `SQLProducer`